src/imeta.cpp

```diff
@@ -2136,6 +2136,7 @@ int display_help(const std::vector<std::string>& _sub_args) {
 
 int main( int argc, const char **argv )
 {
+    set_ips_display_name("imeta");
     signal( SIGPIPE, SIG_IGN );
 
     utils::set_ips_display_name(boost::filesystem::path{argv[0]}.filename().c_str());
```

Comment:
Hmm... what's going on here?

Reply:
oh, yes, that does appear to be.... exactly what we want already?

Reply:
The existing utils works just fine. Any preferences? If not, think we'll keep the old one.

Reply:
Turns out, this was fixed in https://github.com/irods/irods/issues/7376 (specifically, https://github.com/irods/irods_client_icommands/pull/426). So we can either remove line 2142 in favor of your change, or just remove your change.

Reply:
Agreed.

Reply:
Removed new line.